Formula/mk.rb

```diff
@@ -0,0 +1,90 @@
+class Mk < Formula
+  include Language::Python::Virtualenv
+
+  desc "Developer friendly build/test CLI launcher"
+  homepage "https://github.com/pycontribs/mk"
+  url "https://files.pythonhosted.org/packages/fe/cd/fe07815319409d39c7b7506e865708caa289a82126f683b63c2e51ff3428/mk-1.0.2.tar.gz"
+  sha256 "6637f035ddf92a43d2daa5264ce3bfbb5c7e937a3967a1618e48f7247a6914d0"
+  license "MIT"
+  revision 1
+
+  depends_on "python@3.9" => :build
```

Comment:
If you're installing into a virtualenv, then Python is a runtime dependency rather than build-only.